examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -7,25 +7,26 @@ import 'package:flutter/scheduler.dart' show timeDilation;
 
 import 'item.dart';
 import 'home.dart';
+import 'updates.dart';
 
 final Map<String, WidgetBuilder> _kRoutes = new Map<String, WidgetBuilder>.fromIterable(
   kAllGalleryItems,
   key: (GalleryItem item) => item.routeName,
-  value: (GalleryItem item) => item.buildRoute
+  value: (GalleryItem item) => item.buildRoute,
 );
 
 final ThemeData _kGalleryLightTheme = new ThemeData(
   brightness: Brightness.light,
   primarySwatch: Colors.lightBlue,
 );
 
-final ThemeData _kGalleryDarkTheme = new ThemeData(
-  brightness: Brightness.dark,
-  primarySwatch: Colors.lightBlue
-);
+final ThemeData _kGalleryDarkTheme =
+    new ThemeData(brightness: Brightness.dark, primarySwatch: Colors.lightBlue);
```

Comment:
If you add a comma after lightBlue it will go back to the old wrapping even when run through dartfmt.